boards/generic_qemuvirt.ld

```diff
@@ -18,6 +18,10 @@ SECTIONS
         *(.rodata*);
     }
 
```

Comment:
We should use this opportunity to also add .bss

Reply:
Fixed